clang/test/CheckedCRewriter/typedefbounds.c

```diff
@@ -0,0 +1,17 @@
+// RUN: cconv-standalone -alltypes -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
```

Comment:
Some sort of merge error here? I think the `CheckedCRewriter` directory doesn't exist anymore.

Reply:
Yes, it's an inevitable consequence of git's design decision not to explicitly track renames (the merits of which may be debated) that it can't protect against this in all cases.  But git does have a heuristic for added files within detected renamed directories that I have seen handle this scenario successfully in the past, and I imagine it would have worked in this case if Aaron had done a normal `git merge` instead of d9b5da666f3403423565f512ef536fad6640d781.